dist/image-sequencer-ui.js

```diff
@@ -484,23 +489,24 @@ function DefaultHtmlStepUi(_sequencer, options) {
                          </div>";
         step.ui.querySelector("div.details").appendChild(div);
       }
-
-      $(step.ui.querySelector("div.details")).append(
+      $(step.ui.querySelector("div.panel-footer")).append(
         '<div class="cal collapse in"><p><button type="submit" class="btn btn-default btn-save" disabled = "true" >Apply</button><span> Press apply to see changes</span></p></div>'
-      );
-
-      
+      );      
+      $(step.ui.querySelector("div.panel-footer")).append(
+        '<button class="pull-right btn insert-step " style="margin-left:10px;margin-top:-40px;border-radius:6px;background-color:#fff;border:solid #bababa 1.1px;" >\
```

Comment:
Lots of inline styles. Please move them to demo.css. also why are you appending this? Can't this be added to the template itself?

Reply:
If i add it to template then It even comes in the load step.

Reply:
Oh ok. Np.

Reply:
But please add the styles to demo.css

Reply:
The line is from the starting i didn't add any css to it.